### DISCUSSION TITLE
Sending PDF through React implementation not working/not clear

### DISCUSSION THREAD

**Discussion by alextownleymakes** [eleith/emailjs#319] [ideas / feature requests]
Hey, this is a very specific use case but I'm working on an app for a client that needs a PDF generated and then emailed. I'm using jsPDF for the PDF generation and that's working great, file is at 23kb so the whole template hits under 50kb, but I can't come up with any way to get it to attach to an email. 

I've paid for the $5/mo version of the software so that I can handle attachments, I have tried many different combinations of configuration in my template, many different configurations of how I'm sending that data through the React front end with both the .send and the .sendForm methods, but no matter what happens I get one of two outcomes: the template populates the email, using the field I've defined under the "attachment" tab and then placed into the email, with either [object Object] (and I understand why this is) OR a huge blob of text that's the base64 string. I've sent that string with the 'header' information (e.g. 'application/pdf') and without, as well as sending it with those used as header info in an object. Ultimately none of that accomplishes anything, it just makes its way thru as a string no matter how it's formatted.

What's the right way to accomplish this? Thanks in advance!

**Comment by zackschuster** [comment]
hi there! i presume you're referring to [this emailjs](https://www.emailjs.com/), which is a completely separate service that just happens to share a name with us. we have no formal connections to that team or their projects; we don't even know if their service uses `emailjs` or e.g. `nodemailer` behind the scenes 😆 unfortunately, that means we can't help you out here. sorry!
